content-repo/extra-docs/articles/troubleshooting.md

```diff
@@ -156,6 +156,9 @@ Screenshot of running a `fetch` command with `debug-mode=true` and the resulting
 ![fetch-incidents-debug](../../../docs/doc_imgs/reference/fetch-incidents-debug.png)
 
 ### Test Integration Module in `debug-mode`
+**Note**: The "Do not use by default" option should be unchecked on the integration instance you are testing. You can uncheck this option by running the 
+`!<instance_name>-test-module debug-mode=true` command.
```

Comment:
how exactly `!<instance_name>-test-module debug-mode=true` will uncheck that option? 
`Do not use by default` is a checkbox in the UI, and can be unchecked from the UI.